src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -149,12 +165,12 @@ class BaseCourseCard extends Component {
       return (
         <div className="ml-auto">
           <Dropdown>
-            <Dropdown.Toggle variant="outline-dark">
-              <FontAwesomeIcon icon={faCog} />
-              <span className="sr-only">
-                course settings for {title}
-              </span>
-            </Dropdown.Toggle>
+            <Dropdown.Toggle
+              as={IconButton}
```

Comment:
Excited to see this change! The ability to use `IconButton` with `Dropdown.Toggle` is a relatively new addition to Paragon 😄